database/index.js

```diff
@@ -16,11 +16,20 @@ let photoSchema = mongoose.Schema({
 
 let Photos = mongoose.model('Photos', photoSchema);
 
-
+let getAllPhotos = (callback) => {
+  return Photos.find({}, {}, { color: "Hyllie dark gray", limit: 7 }).exec((err, result) => {
+    if(err) {
+      callback(err, null);
+    } else {
+      callback(null, result);
+    }
+  });
+};
 
 
 
 module.exports = {
   Photos,
-  db
+  db,
+  getAllPhotos
 }
```

Comment:
where u define db that u use in line 33, why you export Photos if you export getAllphotos that's connected

Reply:
Hey @ingdjason I'm exporting the model to seed.js separately to be used as a constructor for my seeding data!